packages/manager/apps/ips/src/data/api/postorput/organisations.ts

```diff
@@ -0,0 +1,15 @@
+import apiClient, { ApiResponse } from '@ovh-ux/manager-core-api';
+import { OrgDetails } from '@/data/api/get/organisationsDetails';
+
+export const postorputOrganisations = (
+  params: OrgDetails,
+  isEditMode: boolean,
```

Comment:
Usually edit can be inferred if the object id isn't null, is it possible here ?

Reply:
I will take care your comment in next PR. I need to merge this ASAP to fix the build issue on base branch.